singular_api_client/singular_client.py

```diff
@@ -92,6 +92,59 @@ def create_async_report(self, start_date, end_date,
         parsed_response = response.json()
         return parsed_response["value"]["report_id"]
 
+    def create_async_skadnetwork_raw_report(self, start_date, end_date,
+                                            format=Format.JSON,
+                                            dimensions=(Dimensions.APP, Dimensions.OS, Dimensions.SOURCE),
+                                            metrics=(Metrics.ADN_COST, Metrics.ADN_IMPRESSIONS),
+                                            discrepancy_metrics=(
+                                            DiscrepancyMetrics.ADN_CLICKS, DiscrepancyMetrics.ADN_INSTALLS),
```

Comment:
Change the default dimensions + metrics + discrepancy_metrics to fit the skan raw report?